recipes/clspv/all/conanfile.py

```diff
@@ -0,0 +1,35 @@
+from conans import ConanFile, CMake, tools
+
+
+class ClspvConan(ConanFile):
+    name = "clspv"
+    version = "0.1"
+    license = "Apache 2.0 license."
+    url = "https://github.com/google/clspv"
+    description = "A prototype compiler for a subset of OpenCL C to Vulkan compute shaders."
+    topics = ("vulkan", "opencl", "spirv", "spir", "vulkan-compute-shaders")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        self.run("git clone https://github.com/daemyung/clspv.git")
```

Comment:
I want to make conandata.yml and use it but there is no public release so I can't write those information. Could you help me get url and checksum?